packages/analytics-js-integrations/src/integrations/GoogleAds/nativeSdkLoader.js

```diff
@@ -1,17 +1,20 @@
+import { DISPLAY_NAME } from '@rudderstack/analytics-js-common/constants/integrations/GoogleAds/constants';
 import { LOAD_ORIGIN } from '@rudderstack/analytics-js-common/v1.1/utils/constants';
-import { logger } from '@rudderstack/analytics-js-common/v1.1/utils/logUtil';
+import Logger from '../../utils/logger';
+
+const logger = new Logger(DISPLAY_NAME);
 
 function loadNativeSdk(sourceUrl) {
   (function (id, src, document) {
-    logger.debug(`in script loader=== ${id}`);
+    logger.info(`in script loader=== ${id}`);
     const js = document.createElement('script');
     js.src = src;
     js.async = 1;
     js.setAttribute('data-loader', LOAD_ORIGIN);
     js.type = 'text/javascript';
     js.id = id;
     const e = document.getElementsByTagName('head')[0];
-    logger.debug('==script==', e);
+    logger.info('script', e);
```

Comment:
Is it required to print the script content every time ?

Reply:
it's there for a long time but agree no need it print script content here. removing it